src/api/index.ts

```diff
@@ -7,7 +7,7 @@ import qs from "qs"
 import merge from 'deepmerge'
 
 const requestInterceptor = async (config: any) => {
-  if (apiConfig.token) {
+  if (apiConfig.token && !config.headers.Authorization) {
```

Comment:
We will not add this check, as when making calls using api method we always want to add the token using the interceptor, but when using client method we do not want to execute this interceptor.

Thus we will update the client method to create a separate instance of axios so that it does not intercept the request using this interceptor and instead directly pass the config that are received in payload.

Reply:
```js
const client = (config: any) => {
  return axios.create().request({ paramsSerializer, ...config })
}
```